### PR TITLE
Update outdated references to Bitergia and PayPal

### PR DESCRIPTION
**Pull request by rrrutledge**
Reference the InnerSource Commons and the ISPO working group instead.

### CODE REVIEW

**Review by jeffabailey — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,15 +16,14 @@ You can find more information about the process in the [contributing section](ht
 
 ## Who is fostering this initiative?
 
-Bitergia, as neutral actor in the software development analytics field aims at
-sharing the knowledge acquire during the last years. This book is currently focused on its main areas of expertise such as the usual metrics and KPI's to use,
+We thank [Bitergia] for starting this book and seeding it with its main areas of expertise such as the usual metrics and KPI's to use,
```
Comment (spier) [inline README.md:19]:
```suggestion
We thank [Bitergia] (especially José Manrique López and Daniel Izquierdo) for starting this book and seeding it with its main areas of expertise such as the usual metrics and KPI's to use,
```

Suggesting to give specific props to these two, who wrote most of the initial book it seems.

File: README.md [hunk 2]
```diff
@@ -39,29 +38,21 @@ See the ["Metrics" section of CONTRIBUTING.md](./CONTRIBUTING.md#metrics).
...
-Thanks for your time here!
+be found at the beginning of each chapter. This initiative is fostered by the [ISPO Working Group].
+For further questions, please contact us in _Slack_!
 
 The book cover was created by [Sebastian Spier](https://spier.hu), using an image by user [Bru-nO](https://pixabay.com/photos/measure-unit-of-measure-meterstab-2737004/), available under the [Pixabay License](https://pixabay.com/service/license/).
 
 All of the content found in this repository is licensed [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/).
+
+[Bitergia]: https://bitergia.com/
+[ISPO Working Group]: https://innersourcecommons.org/community/#ispo
```
Comment (spier) [inline README.md:57]: This deep link does not work. i.e. it does not jump to the ISPO section on that page.

File: README.md [hunk 3]
```diff
@@ -39,29 +38,21 @@ See the ["Metrics" section of CONTRIBUTING.md](./CONTRIBUTING.md#metrics).
 
 ## References
 
-For further reading, we encourage you to participate in the [InnerSource Commons site](https://paypal.github.io/InnerSourceCommons/)
-funded by PayPal where dozens of companies are already sharing experiences and working to
```
Comment (spier) [inline README.md:43]: When reading this, I was wondering if somewhere on https://innersourcecommons.org/ we should have a history section that acknowledges the companies that have enabled the birth and the initial growth of the ISC?

As eventually all of the living resources will have their references to PayPal removed, which seems a bit unfair.

File: README.md [hunk 4]
```diff
@@ -39,29 +38,21 @@ See the ["Metrics" section of CONTRIBUTING.md](./CONTRIBUTING.md#metrics).
...
-although given some technical issues, we finally decided to open this under a personal
-repository.
-
-For further questions, please contact José Manrique López or Daniel Izquierdo:
-(jsmanrique|dizquierdo) at bitergia dot com.
-
-
-Thanks for your time here!
+be found at the beginning of each chapter. This initiative is fostered by the [ISPO Working Group].
+For further questions, please contact us in _Slack_!
```
Comment (spier) [inline README.md:50]: Do we want to link to our slack?

```suggestion
For further questions, please contact us in [Slack](https://innersourcecommons.org/slack)!
```

**Review by spier — CHANGES_REQUESTED** [review]
Good changes overall.

Left some comments.

File: README.md (hunk 2 above)
Comment (spier) [reply README.md:57]: Found the answer to this myself :)

File: README.md (hunk 1 above)
Comment (rrrutledge) [reply README.md:19]: Great idea!

**Review by spier — APPROVED** [review]

**Pull request merged**